ynr/apps/people/tests/test_person_update.py

```diff
@@ -547,3 +546,17 @@ def test_biography_last_updated_timestamp(self):
             response,
             "This statement was last updated on {}.".format(last_updated),
         )
+        # make a second edit to the same person and ensure
+        # the biography timestamp from the previous edit is still shown
+        form["name"] = "Bess Palmer"
+        form["source"] = "Mumsnet"
+        form.submit()
+        self.person.refresh_from_db()
+
+        self.assertEqual(len(self.person.versions), 2)
+        self.assertEqual(self.person.name, "Bess Palmer")
+        self.assertNotEqual(self.person.biography_last_updated, None)
+        self.assertContains(
+            response,
+            "This statement was last updated on {}.".format(last_updated),
+        )
```

Comment:
I don't think this is doing quite what you want. The response object referenced by the `response` variable hasn't changed from when it was last checked. So you need to make to update the object that the var is pointing at before running this test. i.e. you need
```
 response = self.app.get(
            "/person/{}/".format(self.person.pk), user=self.user
        )
```
before line 559.